docs/en/ingest-management/security/fleet-roles-and-privileges.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[fleet-roles-and-privileges]]
+= Required roles and privileges
+
+Beginning with {stack} version 8.1, you no longer require the built-in `elastic` superuser credentials to use {fleet} and integrations.
+
+Assigning the {kib} feature privileges `Fleet` and `Integrations` grants access to these features:
+
+`all`:: Grants full read-write access.
+`read`:: Grants read-only access.
+
+The built-in `editor` role grants the following privileges, supporting full read-write access to {fleet} and integrations:
```

Comment:
Should this be capitalized? `Integrations` instead of `integrations`.
Same consideration below, on line 16, 21 and 27

Reply:
Thanks Cristina! I think it should be capitalized so I've fixed all instances: `Integrations`.